roles/system/templates/deployments/system-app-deployment.yaml

```diff
@@ -0,0 +1,450 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: system-app
+  namespace: "{{ meta.namespace }}"
+  labels:
+    app: 3scale-api-management
+    threescale_component: system
+    threescale_component_element: app
+spec:
+{% if app.hpa.enabled is defined and app.hpa.enabled == false %}
+  replicas: {{ app.replicas | default(app_replicas) }}
+{% endif %}
+  selector:
+    matchLabels:
+      app: 3scale-api-management
+      threescale_component: system
+      threescale_component_element: app
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  template:
+    metadata:
+      labels:
+        app: 3scale-api-management
+        threescale_component: system
+        threescale_component_element: app
+    spec:
+      volumes:
```

Comment:
Normally we place volumes at the end of the object definition, isn't it?

Reply:
Moved in https://github.com/3scale/saas-operator/pull/43/commits/102f64f1a4091bf1c9a90a9331ca5eb759cf3034